src/mbfxml2ex.py

```diff
@@ -825,20 +837,21 @@ def load(region, data, options):
         merge_fields_with_nodes(field_module, node_identifiers, field_info)
         element_ids = create_elements(field_module, connectivity, field_names=['coordinates', 'radius', 'rgb'])
         if group_name is not None:
-            create_group(field_module, group_name, element_ids)
+            create_group_elements(field_module, group_name, element_ids)
         if anatomical_name is not None:
-            create_group(field_module, anatomical_name, element_ids)
+            create_group_elements(field_module, anatomical_name, element_ids)
         if element_ids and is_option('external_annotation', options):
             if options['external_annotation'] and 'anatomical term' in tree:
                 print('create external annotation for tree', tree['anatomical term'])
     for contour in data.get_contours():
         connectivity = determine_contour_connectivity(contour['data'], contour['closed'])
         node_identifiers = create_nodes(field_module, contour['data'])
-        field_info = {'rgb': contour['rgb'], 'annotation': contour['name']}
+        field_info = {'rgb': contour['rgb']}  # , 'annotation': contour['name']}
```

Comment:
Remove the commented out code if it is going.

Reply:
Done